root/entity/edits.tt

```diff
@@ -12,6 +12,7 @@
     SET layout_title = all_edits ? l('Edits for {name}', { name => name_for_title }) : l('Open Edits for {name}', { name => name_for_title });
     SET heading = all_edits ? l('Edits for {name}', { name => link_for_heading }) : l('Open Edits for {name}', { name => link_for_heading });
     SET subheading = html_escape(entity.l_type_name or format_entity_type_name(ent_type));
+    SET subheading = l('{series_type} series', { series_type => subheading }) IF ent_type == 'series';
```

Comment:
It is not consistent with subheading for other series pages at https://github.com/metabrainz/musicbrainz-server/blob/7d2a56fe1bd8a4f6b691aab72a177768db4e7439/root/series/SeriesHeader.js#L24.

Reply:
It's not, but it wasn't before either - series pages show just Series with the type on the sidebar, while the edit list does not have a sidebar to show the type. Do you feel we should just show "series" on all? Or show the type on all?

Reply:
Either way, just make sure it can be localized correctly. `lp('Series', 'singular')` is already localized while `'{series_type} series'` is probably not localizable. It is composed of the singular `{series_type}` whereas plural term is required in many/most languages.

Reply:
Specifying the series type in the subheader seems like a nice change to me, though I'd show it on all (change the series header yvanzo linked to, too).

For proper localization you may need a macro that specifies each string separately, something like this:

```
[%~ MACRO series_subheading(type) BLOCK;
    IF type = 'artist';
        lp('Artist series', 'singular');
    ELSIF type = 'event':
        lp('Event series', 'singular');
    # ... etc.
    END;
END -%]
```

Reply:
My only problem with that is needing to update it every time we add a new series type, but I guess we can live with that :) Will do.

Reply:
You're right, though there are other spots we use this technique and we really only have to update them every $x years, so it's manageable. :P

Reply:
Alternatively, you may directly use the name of series type if it can be changed as follows:
- _Catalogue_ :ok:,
- _Event_ → _Event series_,
- _Festival_ :ok:,
- _Recording_ → _Recording series_,
- …

Series type names would be more consistent if they always designate the same thing: here the series type rather than the series item type. It would be consistent with other entity type attributes too.

However, such change probably has larger implications.